base/BUILD

```diff
@@ -75,3 +75,36 @@ structure_test(
     config = "testdata/certs.yaml",
     image = ":check_certs_image",
 )
+
+# For convenience, rename busybox-x86_64 to busybox.
+genrule(
+    name = "busybox_tar",
+    srcs = ["@busybox//file"],
+    outs = ["busybox.tar"],
+    cmd = "mkdir busybox; cp $(<) busybox/busybox; cd busybox; \
+           for cmd in $$(busybox --list); do \
+             ln -s busybox $$cmd; \
+           done; \
+           cd ..; \
+           tar -cf $@ busybox",
+)
+
+# A debug image with busybox available.
+docker_build(
+    name = "debug",
+    base = ":cc",
+    directory = "/",
+    entrypoint = ["sh"],
+    env = {
+        "PATH": "/busybox",
```

Comment:
That doesn't seem to work, but maybe I'm missing something?

```shell
docker run -it --entrypoint="/busybox/busybox" bazel/busybox:debug sh
/ # echo $PATH
${PATH):/busybox
```

I tried with parens and with nothing...